src/screens/Messages/List/index.tsx

```diff
@@ -171,8 +172,17 @@ function ChatListItem({chat}: {chat: TempDmChatDefs.ChatView}) {
   const {getAgent} = useAgent()
 
   let lastMessage = _(msg`No messages yet`)
+  let lastMessageSentAt = null
   if (TempDmChatDefs.isMessageView(chat.lastMessage)) {
-    lastMessage = chat.lastMessage.text
+    if (chat.lastMessage.sender?.did === getAgent().session?.did) {
```

Comment:
No reading mutable stuff during render pls. Use `currentSession` from `useSession`.

Reply:
Ohh ok I'll have to change that in a few places